FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -67,11 +67,6 @@ struct Dashboard: View {
         .onAppear() {
             AnalyticsManager.shared.trackPageview(path: "/")
 
-            // TODO: refresh if issues are old too?
-            if store.state.issues.isEmpty {
```

Comment:
So the `active` scene phase change is always triggered upon launch?

Reply:
Yep, active happens when an app becomes frontmost and interactive, so the common cases would be startup and returning from the background. It can happen pretty rapidly when multitasking but I think that's OK given the timer on fetching issues again